testing/generate-mock-info

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+import unittest.mock
+
+import mock
+
+
+def main() -> int:
+    mock_all = mock.__all__  # type: ignore
+    unittest_mock_all = unittest.mock.__all__  # type: ignore
+    mock_version = mock.__version__  # type: ignore
```

Comment:
want to make a typeshed PR for this?

Reply:
https://github.com/python/typeshed/pull/3923